website/content/en/docs/building-operators/ansible/development-tips.md

```diff
@@ -64,7 +64,7 @@ this, initialize a new project:
 
 ```bash
 $ mkdir foo-operator && cd foo-operator
-$ operator-sdk init --plugins=ansible --domain=example.com --group=foo --version=v1alpha1 --kind=Foo --generate-role
+$ operator-sdk init --plugins=ansible --domain=example.com --group=foo --version=v1 --kind=Foo --generate-role
```

Comment:
The other tutorials use `v1alpha1`. Why change this only for ansible?

Reply:
because i was only changing what i touched. do you want me to change it everyone else, too? i think we should, if that's the recommended style now.

Reply:
Is that the recommended style? Why not `v1alpha1`? If I'm creating a project for the first time, I don't expect my APIs to be stable already.

Reply:
should be nice we check the others as well. are we using v1 or v1alpha1? As the samples too.

Reply:
removed these changes